tokio/src/fs/read_dir.rs

```diff
@@ -20,12 +20,15 @@ pub async fn read_dir(path: impl AsRef<Path>) -> io::Result<ReadDir> {
     Ok(ReadDir(State::Idle(Some(std))))
 }
 
-/// Stream of the entries in a directory.
+/// Read the the entries in a directory.
 ///
-/// This stream is returned from the [`read_dir`] function of this module and
-/// will yield instances of [`DirEntry`]. Through a [`DirEntry`]
-/// information like the entry's path and possibly other metadata can be
-/// learned.
+/// This struct is returned from the [`read_dir`] function of this module and
+/// will yield instances of [`DirEntry`]. Through a [`DirEntry`] information
+/// like the entry's path and possibly other metadata can be learned.
+///
+/// A `ReadDir` can be turned into a `Stream` with [`ReadDirStream`].
+///
+/// [`ReadDirStream`]: ../../tokio_stream/wrappers/struct.ReadDirStream.html
```

Comment:
> ../../tokio_stream/wrappers/struct.ReadDirStream.html

Does this path also work if uploaded to docs.rs?

Reply:
Hmm, I initially thought it would because `tokio-macros` does it, but that only works because it is re-exported into Tokio, and it doesn't work on `tokio-macros` directly.